frontend/src/metabase/components/ModalContent/ModalContent.styled.ts

```diff
@@ -13,7 +13,6 @@ export const ActionsWrapper = styled.div`
 export const ModalContentActionIcon = styled(Icon)`
   color: var(--mb-color-text-light);
   cursor: pointer;
-  padding: 0.5rem;
```

Comment:
The confirmation modal in Admin / Performance still has a missing close button. It comes back if I add `padding: 0;` here. Worth doing?

Reply:
I was not able to replicate this
![Screen Shot 2025-02-28 at 1 21 44 PM](https://github.com/user-attachments/assets/80bc32ef-0061-45cc-89e1-6740fdff0220)